lms/djangoapps/instructor/views/api.py

```diff
@@ -2141,6 +2143,28 @@ def get_course_forums_usage(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+def graph_course_forums_usage(request, course_id):
+    """
+    Generate a d3 graphable csv-string by checking the report store for the clicked_on file
+    """
+    clicked_text = request.GET.get('clicked_on')
+    course_id = CourseKey.from_string(course_id)
```

Comment:
Is this returning a course key? If so, would a different name be better for the var?